postfix/hooks/init

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# setup user/group
+id -u {{cfg.mail_owner}} &>/dev/null || useradd {{cfg.mail_owner}}
```

Comment:
Not a blocker, but I think in the future we'll want to express this user/group requirement in the plan.  Plan files don't currently have this capability,  but it is something we're aware of in the context of software that requires [privilege drops ](https://github.com/habitat-sh/habitat/issues/4373)(nginx/apache/etc).  We'll **probably** want to try to make it applicable to this use case as well.